vllm/v1/core/scheduler_output.py

```diff
@@ -4,6 +4,8 @@
 from typing import TYPE_CHECKING, Dict, List, Optional, Set, Tuple
 
 if TYPE_CHECKING:
+    import numpy as np
```

Comment:
ditto. I think it's better to just import it outside the guard.

Reply:
done